README.md

```diff
@@ -0,0 +1 @@
+This Repository is A Bot that is written in the Sapphire FrameWork 
```

Comment:
Can we expand on this README?
Things like:
- Why should you use it?
- Description of what the bot does